tests/functional/test_libraries.py

```diff
@@ -113,12 +113,16 @@ def test_gevent_monkey(pyi_builder):
 
 
 @xfail(is_darwin, reason='Issue #1895.')
+@importorskip('tkinter')
+@importorskip('Tkinter')
```

Comment:
This is just a temporary patch (on its own commit) to work around the absence of tkinter in the official python images on docker hub. I opened an issue with them to figure out whether tkinter is intentionally omitted: https://github.com/docker-library/python/issues/122

I'm waiting to see how they respond to determine the right fix.